template-only-bin/install-template.sh

```diff
@@ -16,9 +16,9 @@ cp -r \
   app \
   docker-compose.yml \
   docs \
-  renovate.json \
   $CUR_DIR
 cd -
 
 echo "Remove files relevant only to template development"
 rm .github/workflows/template-only-*
+rm -rf docs/decisions/template
```

Comment:
nit add a newline at end of file

Reply:
Is GitHub hiding it or am I doing something dumb? 

<img width="583" alt="CleanShot 2023-06-13 at 08 50 42@2x" src="https://github.com/navapbc/template-application-nextjs/assets/371943/f731d7a3-4492-4c32-b186-aeeb1dc894df">